llvm/lib/Target/RISCV/RISCVISelLowering.cpp

```diff
@@ -4376,7 +4382,7 @@ static SDValue lowerBitreverseShuffle(ShuffleVectorSDNode *SVN,
 
 // Given a shuffle mask like <3, 0, 1, 2, 7, 4, 5, 6> for v8i8, we can
 // reinterpret it as a v2i32 and rotate it right by 8 instead. We can lower this
-// as a vror.vi if we have zvbb, or otherwise as a vsll, vsrl and vor.
+// as a vror.vi if we have zvkb, or otherwise as a vsll, vsrl and vor.
```

Comment:
I'm sure we're inconsistent about it, but can we try to write extension names with capital Z to match their official naming. Can update that on the lines you're touching.